webpack.config.js

```diff
@@ -85,6 +95,7 @@ const config = {
   optimization: {
     // Workaround for https://github.com/webpack/webpack/issues/7760
     usedExports: false,
+    minimizer: [new OptimizeCSSAssetsPlugin({})],
```

Comment:
@julienw  I think issue only mentioned about minimizing CSS. Therefore is it okay If I address minimizing JS in this PR?

Reply:
Previously there were no `minimizer`, this means that Webpack automatically minimizes JS. Because in this PR you sets a `minimizer` this overrides webpack's default behavior and so JS isn't minimized anymore. That's why it needs to be added as well: so that the existing behavior still runs.
I hope it's clear!